src/neo-vm/Types/Integer.cs

```diff
@@ -10,36 +10,32 @@ public class Integer : PrimitiveType
     {
         public const int MaxSize = 32;
 
-        private readonly int _length;
         private readonly BigInteger value;
 
+        public override ReadOnlyMemory<byte> Memory => value.IsZero ? ReadOnlyMemory<byte>.Empty : value.ToByteArray();
```

Comment:
we can cache the result

Reply:
I don't want to cache it because it will double the space.